src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -58,15 +58,27 @@ def _get_start_and_end_indices(data: list[TimeSeriesPoint]) -> tuple[int, int]:
     return start, end
 
 
-def handle_send_historical_data_to_seer(alert_rule: AlertRule, project: Project, method: str):
+def handle_send_historical_data_to_seer(
+    alert_rule: AlertRule,
+    snuba_query: SnubaQuery,
+    project: Project,
+    method: str,
+    event_types: list[SnubaQueryEventType.EventType] | None = None,
+):
     try:
         rule_status = send_historical_data_to_seer(
             alert_rule=alert_rule,
             project=project,
+            snuba_query=snuba_query,
+            event_types=event_types or snuba_query.event_types,
```

Comment:
`event_types` is updated in a separate table so we can't add it to `updated_query_fields` like the other stuff